wremnants/datasets/datasets2016_v9.py

```diff
@@ -0,0 +1,164 @@
+import narf
+import logging
+import subprocess
+import glob
+import pathlib
+
+lumicsv = f"{pathlib.Path(__file__).parent.parent}/data/bylsoutput.csv"
+lumijson = f"{pathlib.Path(__file__).parent.parent}/data/Cert_271036-284044_13TeV_Legacy2016_Collisions16_JSON.txt"
+
+#TODO add the rest of the samples!
+def makeFilelist(paths, maxFiles=-1):
+    filelist = []
+    for path in paths:
+        filelist.extend(glob.glob(path) if path[:4] != "/eos" else buildXrdFileList(path, "eoscms.cern.ch"))
+    return filelist if maxFiles < 0 else filelist[:maxFiles]
+
+def getDatasets(maxFiles=-1, filt=None, mode=None):
```

Comment:
Would be good to separate the datasets and the function calls rather than repeating everything

Reply:
yes..In that case the actual mapping of the datasets and the attributes can go into a dictionary.